entity-framework/core/what-is-new/ef-core-5.0/breaking-changes.md

```diff
@@ -385,6 +446,32 @@ Provider-specific methods map to a database function. The computation done by th
 
 Since there's no way to mimic behavior of database functions accurately, you should test the queries containing them against same kind of database as in production.
 
+<a name="getcolumnname-obsolete"></a>
+
+### IProperty.GetColumnName() is now obsolete
+
+[Tracking Issue #2266](https://github.com/dotnet/efcore/issues/2266)
+
+#### Old behavior
+
+`GetColumnName()` returned the name of the column that a property is mapped to.
+
+#### New behavior
+
+`GetColumnName()` still returns of a column that a property is mapped to, but this behavior is now ambiguous since EF Core 5 supports TPT and simultaneous mapping to a view or a function where these mappings could use different column names for the same property.
```

Comment:
```suggestion
`GetColumnName()` still returns the name of a column that a property is mapped to, but this behavior is now ambiguous since EF Core 5 supports TPT and simultaneous mapping to a view or a function where these mappings could use different column names for the same property.
```